### PR TITLE
fix: Update surface level 2 to be grey200

### PR DESCRIPTION
**Pull request by alexandra-lim**
Screen Shot 2022-08-16 at 10 14 08 AM

Forgot to update this when I added the system and interactiveSurface tokens

### CODE REVIEW

File: data/light.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         "type": "color"
       },
       "level2": {
-        "value": "{colors.grey.100}",
+        "value": "{colors.grey.200}",
```
Comment (alexandra-lim) [inline data/light.json:31]: This is the one change here

**Comment by iryanclarke** [comment]
I also borked this MR with prettier 😬

**Review by CCAyl — APPROVED** [review]
beauty

**Review by iryanclarke — APPROVED** [review]
:shipit:

**Pull request merged**